src/kube-runtime/src/runtime

```diff
@@ -39,15 +40,15 @@ PAI_RUNTIME_LOG_FILE=${PAI_LOG_DIR}/user.pai.all
 
 # comment for the script purpose
 # priority=value
-# ${PAI_RUNTIME_DIR}/runtime.sh >> ${PAI_RUNTIME_LOG_FILE} 2>&1
+# ${PAI_RUNTIME_DIR}/runtime.sh >> ${PAI_USER_LOG_FILE} 2>&1
```

Comment:
Change this comment? The output should redirect to the runtime log

Reply:
Will remove this. Use plugins to add pre/post commands instead of write it here.